src/Core/Localization/CLDR/Locale.php

```diff
@@ -118,6 +125,7 @@ public function __construct(LocaleData $localeData)
         $this->percentPatterns = $localeData->getPercentPatterns();
         $this->currencyPatterns = $localeData->getCurrencyPatterns();
         $this->currencies = $localeData->getCurrencies();
+        $this->territories = $localeData->getTerritories() ?: [];
```

Comment:
Why does this one need a special case? (compared to all other data) Can't it remain `null` ?